cekit/generator/base.py

```diff
@@ -60,11 +61,18 @@ def __init__(self, descriptor_path, target, overrides):
 
                 LOGGER.debug("Loading override '{}'".format(override))
 
-                override_artifact_dir = os.path.dirname(os.path.abspath(override))
-                if not os.path.exists(override):
-                    override_artifact_dir = os.path.dirname(os.path.abspath(descriptor_path))
-                self._overrides.append(Overrides(tools.load_descriptor(
-                    override), override_artifact_dir))
+                # HTTP Handling
+                if override.startswith('http'):
```

Comment:
I wonder if we need to do this if we have the `urlparse()` mechanism which provides `scheme`.

Reply:
Yeah I think its a toss-up which to go with:
```
parsed_url = urlparse(url)
if parsed_url.scheme in ['http', 'https']:
```
or
```
if override.startswith('http'):
```

Actually one potentially benefit could be to handle `file://` URLs as well so if we do:
```
parsed_url = urlparse(url)
if parsed_url.scheme:
```

Reply:
Yah, I think it would make sense to refactor it to use the same mechanism. Or just use the `download` (possibly rename?) method every time. But this would require some more refactoring I guess.

Reply:
Done